src/components/TaskClusterMap/TaskClusterMap.js

```diff
@@ -377,8 +392,48 @@ export class TaskClusterMap extends Component {
             }}
           />
         }
+        {!!this.props.mapToLarge && !this.state.locatingToUser &&
```

Comment:
Typo in `Too` -- maybe we can rename to better reflect that really the map is zoomed too far out rather than simply too large?